src/api/routers/auth.py

```
@@ -0,0 +1,541 @@
+
+    webauthn_svc = WebAuthnService(session, rp_id=settings.webauthn_rp_id or "localhost", rp_name=settings.webauthn_rp_name or "AgentJob")
+    token_svc = TokenService(session, secret_key=settings.jwt_secret)
+    auth_svc = AuthService(session, secret_key=settings.jwt_secret)
+    device_svc = DeviceTrustService(session)
+
+    if not body.username:
+        raise HTTPException(status_code=400, detail="需要用户名")
+
+    # Look up user
+    result = await session.execute(
+        select(UserModel).where(UserModel.username == body.username)
```

Pylint (4.0.5)

[error] Undefined variable 'UserModel'

(E0602)